tests/models/language.py

```diff
@@ -17,3 +20,143 @@ def test_language_repr():
     assert (
         "<Language: %s>" % language.name
         == repr(language))
+
+
+@pytest.mark.django_db
+def test_unused_language_is_not_live():
+    unused_language = LanguageDBFactory()
+    assert unused_language.translationproject_set.count() == 0
+    assert unused_language not in Language.live.all()
+
+
+@pytest.mark.django_db
+def test_language_only_in_disabled_proj_is_not_live():
+    language = LanguageDBFactory()
+    project = ProjectDBFactory(source_language=language)
```

Comment:
It fails that way. But found another way to get it working with less queries and that is considerably faster.